Kconfig

```diff
@@ -24,6 +24,12 @@ config LP_MEMORY_BANKS
 	help
 	  Available memory banks count for Low Power memory
 
+config CONFIG_LP_SRAM
```

Comment:
config LP_SRAM

Reply:
Why its not "depends on CAVS" if LP_MEMORY_BANKS depends on CAVS?

Anyway the best option:
LP_SRAM should depends on CAVS and LP_MEMORY_BANKS depends on LP_SRAM